scripts/tests/run_java_test.py

```diff
@@ -87,10 +88,18 @@ def main(app: str, app_args: str, tool_path: str, tool_cluster: str, tool_args:
         logging.info("Testing pairing cluster")
 
         test = CommissioningTest(log_cooking_threads, log_queue, command, tool_args)
-        controller_exit_code = test.RunTest()
-
-        if controller_exit_code != 0:
-            logging.error("Test script exited with error %r" % test_script_exit_code)
+        try:
+            test.RunTest()
+        except Exception:
+            sys.exit(1)
```

Comment:
Please print out the exception as well, otherwise we silently exit with 1. Same below